### DISCUSSION TITLE
View table relationships

### DISCUSSION THREAD

**Discussion by gavindew** [open-metadata/OpenMetadata#11712] [Ideas]
The Table API has the ability to load foreign key relationships, however, this information does not display on the catalog. It would great if the relationships to / from a table could be displayed with the ability to navigate between tables by clicking on the relationships. Something like https://dbdocs.io/Holistics/Ecommerce?schema=ecomm_core&view=table_structure&table=product_tags

**Comment by christianneureiter** [comment]
I like the idea, I think there are some more "out of the box" source information which can be used in OM.

**Comment by Prajwal214** [comment]
We have a feature for ER Diagrams, but it is available only for Collate SaaS.
